### ISSUE TITLE
Error message with /e:off (again)

### ISSUE THREAD

**Issue by Gei0r** [chrisant996/clink#570]
Hi, thanks for fixing #569 so quickly. With the new version `1.6.6.87eebc`, the command with `/e:off` is run sucessfully, however an error message is printed:

```
$ cmd.exe /e:off /c "echo a"
Das Sprungziel - eof wurde nicht gefunden.
a
```
"The jump destination - eof was not found"

**Comment by Gei0r** [comment]
Adding a line:
```
:eof
```
At the end of `clink.bat` seems to fix this.

**Comment by chrisant996** [comment]
Can I ask, what is the reason for turning off command extensions?  It seems like a lot of things will break.

Anyway, I'll look into it.  But I'm surprised that I didn't encounter that problem when I tested the first fix.

**Comment by Gei0r** [comment]
> Can I ask, what is the reason for turning off command extensions? It seems like a lot of things will break.

Unfortunately I don't know. It's in some old codebase I inherited. The commit introducing this line is from 2006 and doesn't have a message or comment.

**Comment by chrisant996** [comment]
- `exit /b` requires command extensions, even though that isn't documented anywhere.
- `cd /d` requires command extensions.

I've updated Clink accordingly, but avoiding `cd /d` is a little messy and inconsistent in Clink (e.g. the user in #333 will have to update their custom script).

I've updated the clink-gizmos and clink-fzf repos accordingly as well to avoid `cd /d`.

**Comment by chrisant996** [comment]
Fixed by 8251455482caca560efeb617aef4a5755b0c13d4.
